dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/validation/DefaultTrackerValidationService.java

```diff
@@ -112,9 +110,10 @@ private ValidationResult validate( TrackerBundle bundle, Validators validators )
         bundle.setEvents( persistables.getEvents() );
         bundle.setRelationships( persistables.getRelationships() );
 
-        reporter.getErrors().addAll( persistables.getErrors() );
-
-        return new ValidationResult().addErrors( reporter.getErrors() ).addWarnings( reporter.getWarnings() );
+        return new Result()
```

Comment:
Maybe now or later 😋 Would be great if `Result` is created in one go without adding to it. For only adding errors and warnings that would be as easy as `new Result(errors, warnings)`. Since we have 2 separate errors `new Result(errors, othererrors, warnings)` looks a bit weird 🤔